enhancements/installer/ipi-aws-china.md

```diff
@@ -0,0 +1,130 @@
+---
+title: ipi-install-aws-china
+authors:
+  - "@wanghaoran1988"
+reviewers:
+  - TBD
+approvers:
+  - TBD
+creation-date: 2020-02-14
+last-updated: 2020-02-14
+status: implementable
+---
+
+# IPI install on AWS China
+
+## Release Sign off Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+We have customers want to install OpenShift on AWS China, but currently
+OpenShift installer doesn't show up the AWS China regions `cn-north-1` and `cn-northwest-1`.
+These two regions are isolated from other global regions and RHCOS AMI pushed to global cannot
+be used in AWS China, [ARNs and api endpoint](https://docs.amazonaws.cn/en_us/aws/latest/userguide/endpoints-arns.html) are different.
+
+Similar to other global regions, we should support these two regions for customer who want deploy
+OpenShift on AWS China.
+
+## Motivation
+
+### Goals
+
+* OpenShift installer support IPI install on AWS China Regions.
+* CI job executing testings on AWS China regions.
+
+### Non-Goals
+
+* It's not a goal to detail how to request and setup a AWS account in AWS China.
+* It's not a goal to detail how to do UPI install.
```

Comment:
I think another goal would be supporting any endpoints for services that are not part of the SDK for those region.